githubmarkdownui/blocks/table.py

```diff
@@ -0,0 +1,40 @@
+from enum import Enum, auto
+from typing import List, Optional
+
+
+class TableAlignmentError(Exception):
+    """Raised when the alignment parameter is not the same length as the number of columns in the table."""
+
+
+class TableDimensionError(Exception):
+    """Raised when each row of the table does not have the same number of columns."""
+
+
+class TableAlignment(Enum):
+    CENTER = auto()
+    LEFT = auto()
+    RIGHT = auto()
+
+
+def table(content: List[List[str]], alignment: List[Optional[TableAlignment]] = None) -> str:
+    """Creates a table using HTML syntax.
+
+    The content parameter is a list containing lists of equal length, which correspond to the contents of the table.
+    The first sublist is the table headers, and each remaining sublist is one row of the table.
+
+    For example, the content parameter [['column 1', 'column 2'], ['hello', 'world'], ['foo', 'bar']] would create a
+    table like this:
+
+    column 1 | column 2
+    ---------|---------
+    hello    | world
+    foo      | bar
```

Comment:
I'd prefer to have an example with alignment param since that's more informative to the readers. I'm good with the current structure.

Reply:
Added an example of right aligning the first column